### PR TITLE
hw 5-3

### PR DESCRIPTION
**Pull request by mikhail771**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -8,7 +16,37 @@
...
+            while (line != null) {
+                result.append(line);
+                line = br.readLine();
+            }
+            words = result.toString().split(" ");
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("Can't read file", e);
+        } catch (IOException exception) {
+            exception.printStackTrace();
+        }
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FileWork.java:37]: do you need two catch block here?

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -8,7 +16,37 @@
...
+
+        for (String word : words) {
+            word = word.toLowerCase().replaceAll("[^A-Za-z]", "");
+            if (word.indexOf(SEARCH_LETTER) == 0) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FileWork.java:41]: we have string method to check if it's starts with some character

Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FileWork.java:41]: let's make this manipulation while reading files. This way you will get rid of using ArrayList

File: src/main/java/core/basesyntax/FileWork.java (hunk 1 above)
Comment (mikhail771) [reply src/main/java/core/basesyntax/FileWork.java:37]: reedLine() method requires second exception

File: src/main/java/core/basesyntax/FileWork.java (hunk 2 above)
Comment (mikhail771) [reply src/main/java/core/basesyntax/FileWork.java:41]: done

**Pull request closed without merging**